lib/template/index.html

```diff
@@ -16,6 +16,7 @@
       repo: ''
     }
   </script>
-  <script src="//cdn.jsdelivr.net/npm/docsify/lib/docsify.min.js"></script>
+  <!-- Docsify v4 -->
+  <script src="//cdn.jsdelivr.net/npm/docsify@4/lib/docsify.min.js"></script>
```

Comment:
as of now, we have 4.11.3 as latest version. 

Can you change that. ?

Reply:
> as of now, we have 4.11.3 as latest version.
> 
> Can you change that. ?

The reason I didn't do that was because the issue mentioned "Major versions". If we hardcode the version number everytime there is a new update, that would be hectic. But regardless I don't mind doing it. 

I was thinking, wouldn't it be best to have both of the script tags generated, and we comment the script with the version, so this way the user has an option, get the latest release all the time or stick with a specific release. Let me know what you think about that.

Reply:
> The reason I didn't do that was because the issue mentioned "Major versions". If we hardcode the version number everytime there is a new update, that would be hectic. But regardless I don't mind doing it.


Yes that would be hectic. I was thinking that we need to release a new version of the cli with update version in the link every time there is a new version for docsify core.




> I was thinking, wouldn't it be best to have both of the script tags generated, and we comment the script with the version, so this way the user has an option, get the latest release all the time or stick with a specific release. Let me know what you think about that.

This is an interesting point. I would appreciate some comments about this in the template. 



Does jsdeliver automatically bumps version to latest minor/patch version when only semver major version is specified ?

Like if the version is set as `4` , does it automatically loads the latest `4.x.x` version ?
cc @jhildenbiddle thoughts here?